### ISSUE TITLE
Create a CHS-jsPsych video-assent plugin

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1329]
# TL;DR

We need to create a CHS-jsPsych video-assent plugin that researchers can use to record older children's agreement to participate.

# Narrative

As a researcher, I need an easy way to include the standard CHS video assent survey, ideally without much custom programming or large chunks of copy/pasted code.

# Acceptance Criteria

- [ ] Tests
- [ ] Documentation
- [ ] Scoping: decide how similar this plugin needs to be to the [EFP video-assent frame](https://lookit.readthedocs.io/projects/frameplayer/en/latest/components/exp-lookit-video-assent/doc.html) in terms of content and functionality
- [ ] Child can agree to participate by selecting from a yes/no multiple choice question, and/or answering verbally with video recording

And depending on scoping decisions:
- [ ] Plugin shows one or more 'pages' of text/images/audio/video/webcam content, which the participant can navigate with back/forward buttons
- [ ] Participant must view every page before they can assent
- [ ] Researcher has the option to record video during the whole procedure or just the last page
- [ ] Conditionally show this trial only to children who meet the age requirement (set via the `minimumYearsToAssent` parameter)
